packages/carbon-web-components/src/components/data-table/table-cell.ts

```diff
@@ -27,10 +38,25 @@ class BXTableCell extends LitElement {
   }
 
   render() {
-    return html` <slot></slot> `;
+    const { disabled, textContent } = this;
+
+    const statusText = ['Starting', 'Active', 'Disabled'];
+    const isStatus = statusText.some((word) => textContent?.includes(word));
+    const isDisabled = textContent!.includes('Disabled');
+    const cellClasses = classMap({
+      [`${prefix}--link`]: isStatus,
+      [`${prefix}--link--disabled`]: disabled || isDisabled,
+    });
```

Comment:
i think the user should be able to use `<cds-link>` here like: https://github.com/carbon-design-system/carbon/blob/33b35df6a77d08d9c25dbf7229b0d7a0feb653f2/packages/react/src/components/DataTable/stories/shared.js#L19. and then this component can stick to just being a simple `slot`